pysass.cpp

```diff
@@ -404,6 +435,116 @@ static void _add_custom_functions(
     sass_option_set_c_functions(options, fn_list);
 }
 
+Sass_Import_List _call_py_importer_f(
+        const char* path,
+        Sass_Importer_Entry cb,
+        struct Sass_Compiler* comp
+) {
+    PyObject* pyfunc = (PyObject*)sass_importer_get_cookie(cb);
+    PyObject* py_path = PyUnicode_FromString(path);
```

Comment:
sweet, this'll use UTF-8 to decode the string: https://docs.python.org/2/c-api/unicode.html#c.PyUnicode_FromString

(I had a concern this would use ascii like `unicode(bytes_object)` would, but it seems the c-api is not always consistent with the python behaviour!)